include/Dialect/Poly/IR/PolyOps.td

```diff
@@ -47,6 +47,8 @@ def Poly_MulConstantOp : Poly_Op<"mul_constant", [ElementwiseMappable]> {
   let results = (outs
     PolynomialLike:$output
   );
+
+  let hasVerifier = 1;
```

Comment:
Consider instead using the trait `AllTypesMatch<["x", "output"]>` which avoids the need for any custom C++

Reply:
Thank you! I was almost sure that `AllTypesMatch` exists )

Reply:
I will update assembly format as well.